src/ui/Quickview/QuickviewDocumentIframe.ts

```diff
@@ -84,6 +84,9 @@ export class QuickviewDocumentIframe {
       sandbox: 'allow-same-origin allow-top-navigation',
       src: 'about:blank'
     });
+    if (this.title) {
```

Comment:
This `if` should not be necessary if `title` is required,

Reply:
Resolved because this option was removed.